roles/installer/vars/main.yml

```diff
@@ -1,7 +1,12 @@
 ---
 # vars file for installer
 pullsecret_file: "{{ dir }}/pull-secret.txt"
-cmd: openshift-baremetal-install
+cmd: >-
+  {%- if release_version is version('4.16', ">=") and ansible_distribution_major_version == '8' -%}
```

Comment:
May user command as var and use ternary so ansible will not complain?

```
.   vars: command: "{{.release_version is version('4.16', ">=") and ansible_distribution_major_version == '8' | ternary('openshift-install', 'openshift-baremetal-install') }}"
ansible.builtin.command: "{{ command }}"
```

Reply:
Thanks, I've modified a bit your suggestion, it should remain clear and readable.